IPython/core/interactiveshell.py

```diff
@@ -258,6 +358,43 @@ class InteractiveShell(SingletonConfigurable):
         """
     ).tag(config=True)
 
+    autoawait = Bool(True, help=
+        """
+        Automatically run await statement in the top level repl.
+        """
+    ).tag(config=True)
+
+    loop_runner_map ={
```

Comment:
Note: For another PR, it would be a good idea to have a way to register new names for this.